### DISCUSSION TITLE
Switch project contribution process from CLA to DCO

### DISCUSSION THREAD

**Discussion by timkpaine** [perspective-dev/perspective#2609] [General]
We are planning to adopt DCO ([Developer Certificate of Origin](https://developercertificate.org)) instead of CLA as the mechanism for authorizing project contributions. More information is available about the differences between these two on the [FINOS Open Source Readiness website](https://osr.finos.org/docs/bok/artifacts/clas-and-dcos).

**There are no changes to the licensing or the copyright ownership for this project.**

Changes are planned for early next week, a PR to the contributing documentation will follow shortly. We will be turning off the CLA enforcement bot and turning on a DCO enforcement bot. Subsequent contributions will need to be signed.

This issue is to provide notice, please provide any comments, questions, or concerns below.

@texodus @sc1f @timkpaine

**Comment by brianblakely** [comment]
Do you know if contributors who are already authorized under the CLA will need to be re-authorized under DCO?

**Comment by timkpaine** [comment]
@brianblakely nope, there is no retroactive change necessary. Only effects future changes.
